helm_deploy/values-dev.yaml

```diff
@@ -82,12 +82,14 @@ generic-service:
     USER-ALLOCATIONS_RULES_LEGACY-ALLOCATOR_ENABLED: false
     USER-ALLOCATIONS_RULES_LEGACY-ALLOCATOR_PRIORITY: -1
     USER-ALLOCATIONS_RULES_ESAP-ASSESSMENTS_ENABLED: true
-    USER-ALLOCATIONS_RULES_ESAP-ASSESSMENTS_PRIORITY: 2
+    USER-ALLOCATIONS_RULES_ESAP-ASSESSMENTS_PRIORITY: 3
     USER-ALLOCATIONS_RULES_ESAP-ASSESSMENTS_ALLOCATE-TO-USER: # TODO: Determine appropriate user
     USER-ALLOCATIONS_RULES_EMERGENCY-ASSESSMENTS_ENABLED: true
-    USER-ALLOCATIONS_RULES_EMERGENCY-ASSESSMENTS_PRIORITY: 1
+    USER-ALLOCATIONS_RULES_EMERGENCY-ASSESSMENTS_PRIORITY: 2
     USER-ALLOCATIONS_RULES_FEMALE-ASSESSMENTS_ENABLED: true
-    USER-ALLOCATIONS_RULES_FEMALE-ASSESSMENTS_PRIORITY: 0
+    USER-ALLOCATIONS_RULES_FEMALE-ASSESSMENTS_PRIORITY: 1
+    USER-ALLOCATIONS_RULES_APPEALED-ASSESSMENTS_ENABLED: true
+    USER-ALLOCATIONS_RULES_APPEALED-ASSESSMENTS_PRIORITY: 0
```

Comment:
Just to be clear on these priorities: we're saying that we apply this rule first? So  the new assessment is always allocated to the arbitrator?

Reply:
Yeah, exactly. Confirmed this with Adam too.